### PR TITLE
quick readme update for file exclusion

### PR DESCRIPTION
**Pull request by agraebe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +222,8 @@ Default: component namespace path + `/**` (e.g. `my/app/**`)
 Glob pattern(s) for files that should be included into the preload.  
 Patterns are based on all available resources (see [resources](#resources)).
 
+**Note:** You can exclude files by prepending the exclamation mark, like so: ```!my/app/ignore.js```.
+
 ### libraries
 
 See [components](#components). Auto mode is looking for `library.js` files.
```
Comment (matz3) [inline README.md]: I think we should also add a note here. As the current default pattern does only include `**/*.fragment.json` files but not `library-preload.json` we don't really need a equivalent exclude for libraries.

File: tasks/preload.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ var maxmin = require('maxmin');
 
 var defaultResourcePatterns = [
 	'**/*.js',
+	'!**/Component-preload.js',
```
Comment (matz3) [inline tasks/preload.js]: I'm not sure if it makes sense to add this exclude to the default patterns as they also apply for `libraries`. In this case you might even include a `Component-preload.js` into your library.

**Comment by matz3** [comment]
When doing the review I again thought about this change and there might be a use case to include `Component-preload.js` files.
- Packaging sub-components into a root component
- Packaging libraries into components or vice versa

I would rather do a check independent of the include/exclude patterns to prevent including an output file (`Component-preload.js` / `library-preload.json`) which is in the same folder as the targeted Component or Library (`Component.js` / `library.js`).

**Comment by matz3** [comment]
@agraebe do you want to follow up on this?
Otherwise I would do a change based on my last comments.

**Comment by agraebe** [comment]
I think we should approach with your suggestions, sorry for the late response!

**Comment by matz3** [comment]
See #25 (documentation update is missing there. I'll add that later). Let me know if this suits your needs.

**Comment by matz3** [comment]
As there was no further feedback I merged #25 which will be included in the next release. I did not adopt the README.md as the behavior should be quite obvious.

**Pull request closed without merging**